packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -32,6 +32,10 @@ class _PointerState {
 
   Offset lastPosition;
 
+  Offset deltaTo(Offset to) {
+    return to - lastPosition;
```

Comment:
this is a minor thing, but fwiw you could write this as:

`Offset deltaTo(Offset to) => to - lastPosition;`